### ISSUE TITLE
Map user-generated text and color styles to plugin output

### ISSUE THREAD

**Issue by nathanacurtis** [EightShapes/specs-plugin#39]
### Problem

The user has their own, already existing styles preferred over the default styles added by the plugin, and would like to map the output to use those styles.

### Solution

Enable a configuration to map custom style names in place of existing styles.

**Comment by thevladimirkrstic** [comment]
Hey @nathanacurtis, any progress on this? 

The quickest solution would be to instruct the users what variables and styles they need to create(how to name them exactly) in a file, and you hard code the exact styles/variables in a plugin to be applied to the generated assets.

**Comment by thevladimirkrstic** [comment]
> First, for paid subscribers, the plugin now already supports aliasing the plugin's variable collection for customizing colors to your own variables you manage separately for your system.

Hmm, I struggle to find how to do that. And yes, I'm a paid user...... Oh, I've found it! This is great and good enough for me! I can alias variables and batch restyle the styles.

P.S. This is the best Figma plugin. I wanted to make documentation for a design system like UI Prep by Molly, but I thought it was all manual work. Last night, I saw Developer Mode updates and thought this might help. Then I saw your post and was blown away when I figured out that it was a plugin. You rock, man!
